src/build.props

```diff
@@ -42,6 +42,7 @@
     <IntermediateOutputPath>$(MsBuildThisFileDirectory)..\bld\obj\$(OutputSubDir)\$(MSBuildProjectName)\</IntermediateOutputPath>
     <OutputPath>$(MsBuildThisFileDirectory)..\bld\bin\$(OutputSubDir)\$(MSBuildProjectName)\</OutputPath>
     <PublishDir>$(OutputPath)\Publish\$(TargetFramework)\</PublishDir>
+    <PublishDir Condition=" '$(RuntimeIdentifier)' != '' ">$(OutputPath)\Publish\$(TargetFramework)\$(RuntimeIdentifier)\</PublishDir>
```

Comment:
>$(OutputPath)\Publish\$(TargetFramework)\ [](start = 59, length = 41)

You could DRY this out by using `$(PublishDir)` here because the properties are evaluated in order.

Reply:
Sure